File: Randy-JRA/Exercise_7.cpp
//Challenge 7 is to Calculate the sum of odd numbers greater than 10 and less or equal than 30

#include<iostream>
using namespace std;
int main()
{
 
	int sum, i;
	
	for(i=11; i<=30; i+=2)
	{
		sum = i+sum;
	} 
	
	cout<<"The sum of odd numbers >10 and <=30 is ="<<sum<<"\n";
}

File: kanye/src/part1/exercise7.rs
// function to print the sum of all even numbers greater than 10 and less than 30
pub fn sum_even_10_30() {
    let mut sum = 0;
    for i in 11..30 {
        if i % 2 == 0 {
            sum += i
        }
    }
    println!("sum of even numbers = {}", sum);
}

// function to print the sum of all odd numbers greater than 10 and less than 30 variant 1
#[allow(dead_code)]
pub fn sum_odd_10_30_1() {
    let mut sum = 0;
    let mut i = 11;
    while i < 30 {
        if i % 2 == 1 {
            sum += i;
        }
    } println!("sum of odd numbers = {}", sum);
}